### DISCUSSION TITLE
Problem di gambar

### DISCUSSION THREAD

**Discussion by TachooDa** [rizkyilhampra/spdhtc#76] [Q&A] [answered]
Misi mas ini kira-kira bisa dijelasin ngga ya, kan saya udah php artisan storage:link tpi buat update gambar masih seperti berikut 
![image](https://github.com/rizkyilhampra/spdhtc/assets/89304676/44c5117a-7b61-4520-82c1-49ca964be6f2)

**Comment by rizkyilhampra** [comment]
Halo

Coba kirim pesan error yang tampil di console browser dan output pada saat menjalankan perintah `php artisan storage:link`.

**Reply by TachooDa** [reply]
kalo yg php artisan storage:link si begini bang
![image](https://github.com/rizkyilhampra/spdhtc/assets/89304676/34eb291d-90de-48cf-b5ac-f96e96919153)

tapi untuk di  console web nya dia kyk gini, jdi dia kayak ngga tmpil di web nya, tpi di database nya masuk/kesimpen
![image](https://github.com/rizkyilhampra/spdhtc/assets/89304676/b66bd937-ff6f-470a-b8c0-0bd64c7057c7)

**Reply by TachooDa** [reply]
klo yg atas kyknya wajar si, soalnya diulang-ulang terus gara-gara penasaran ko dia storage nya ngga mao gitu buat digambar nya doang

**Reply by rizkyilhampra** [reply]
Coba ss tampilan di folder `C:\xampp\htdocs\sppg\public` ada symbolic link ke folder storage atau tidak? jika ada, coba buka apakah terdapat gambar disana?

**Reply by TachooDa** [reply]
bentar bang

**Reply by TachooDa** [reply]
dia begini

**Reply by TachooDa** [reply]
![image](https://github.com/rizkyilhampra/spdhtc/assets/89304676/3bef4e59-7e89-4705-9089-5764f530ab38)

**Reply by TachooDa** [reply]
apa harus ganti lokasi disk nya ya, tapi dari awaln masih default yng "s3"

**Reply by rizkyilhampra** [reply, marked as answer]
Coba hapus folder/file storage nya terlebih dahulu, kemudian lakukan `php artisan storage:link` lagi di path `C:\xampp\htdocs\sppg`. Jika masalah masih berlanjut coba gunakan terminal dari `Git bash` bukan dengan `Powershell`

**Reply by TachooDa** [reply]
oke bentar bang

**Reply by rizkyilhampra** [reply]
> apa harus ganti lokasi disk nya ya, tapi dari awaln masih default yng "s3"

https://github.com/rizkyilhampra/spdhtc/blob/0952e4060474191475390d302fa48b880bd55578/.env.example#L33

Default tetap `local`. Jika mengikuti tutorial install di `README.md` harusnya tidak ada masalah

**Reply by TachooDa** [reply]
ahhh finally bang bisa timbul
![Uploading image.png…]()

**Reply by TachooDa** [reply]
udah timbul si bang
![image](https://github.com/rizkyilhampra/spdhtc/assets/89304676/05cef4ba-b150-4590-b452-5a1e9733cef5)

**Reply by TachooDa** [reply]
ternyata saya ngga merhatiin bahwa pas saya ekstrak  zip dia udah ada folder storage nya

**Reply by TachooDa** [reply]
btw-btw mao izin buat eksplor sama buat tugas akhir bang? lgi belajar laravel juga soalnya
aman ngga kira-kira? heheh

**Reply by rizkyilhampra** [reply]
> btw-btw mao izin buat eksplor sama buat tugas akhir bang? lgi belajar laravel juga soalnya aman ngga kira-kira? heheh

Silahkan, lihat bagian lisensi https://github.com/rizkyilhampra/spdhtc?tab=WTFPL-2-ov-file . Janga lupa ***star*** repo nya ✌️
